### ISSUE TITLE
Multi-column text editor padding/margins apply to each column separately

### ISSUE THREAD

**Issue by RichardNesbitt** [elementor/elementor#18794]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

If I set a text editor to have multiple columns, and then apply left/right margin or padding to it in the Advanced tab, it gets applied to each column, not the whole block.

### Steps to reproduce

Place a text editor block. Set to 3 columns. Enter enough text to populate all 3 columns. Add left/right margin to the block. See it apply to all 3

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.35
	PHP Version: 8.1.6
	PHP Memory Limit: 512M
	PHP Max Input Vars: 5000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0
	Site URL: http://toi.dev.r2az.com
	Home URL: http://toi.dev.r2az.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: richardknesbitt@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: Astra Child
	Version: 1.0.0
	Author: Brainstorm Force
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.8.1
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (iPhone; CPU iPhone OS 15_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/102.0.5005.67 Mobile/15E148 Safari/604.1

== Active Plugins ==
	Astra Pro
		Version: 3.6.8
		Author: Brainstorm Force

	Elementor
		Version: 3.6.5
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.1
		Author: Elementor.com

	Ultimate Addons for Elementor
		Version: 1.36.6
		Author: Brainstorm Force

	WooCommerce
		Version: 6.5.1
		Author: Automattic

== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Active by default
	Hide native WordPress widgets from search results: Active by default
	: Inactive by default
	Flexbox Container: Inactive by default
	Default to New Theme Builder: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default

== Log ==
	
PHP: showing 7 of 7PHP: 2022-05-25 20:30:16 [notice X 617][/home/toi/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/toi/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/toi/public_html/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/toi/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /home/toi/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /home/toi/public_html/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/toi/public_html/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /home/toi/public_html/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(236): ElementorPro\Plugin -> autoload()
',
)]
PHP: 2022-05-26 07:13:30 [notice X 1][/home/toi/public_html/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /home/toi/public_html/wp-content/plugins/elementor-pro/core/utils/collection.php(11): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /home/toi/public_html/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/home/toi/public_html/wp-content/plugins/elementor-pro/core/utils/collection.php',
)
#2: /home/toi/public_html/wp-content/plugins/elementor-pro/core/database/query-builder.php(1222): ElementorPro\Plugin -> autoload()
#3: /home/toi/public_html/wp-content/plugins/elementor-pro/core/database/query-builder.php(1081): ElementorPro\Core\Database\Query_Builder -> get_bindings()
#4: /home/toi/public_html/wp-content/plugins/elementor-pro/core/database/model-query-builder.php(89): ElementorPro\Core\Database\Query_Builder -> get()
',
)]
PHP: 2022-05-27 02:35:12 [warning X 3][/home/toi/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-27 02:42:39 [warning X 2][/home/toi/public_html/wp-content/plugins/elementor/includes/managers/image.php::81] Undefined array key 2 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-27 02:43:39 [notice X 111][/home/toi/public_html/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-27 02:49:36 [error X 1][/home/toi/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::294] Uncaught Error: Call to a member function update_meta() on bool in /home/toi/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:294
Stack trace:
#0 /home/toi/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(167): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home/toi/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home/toi/public_html/wp-includes/class-wp-hook.php(307): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home/toi/public_html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters('', Array)
#5 /home/toi/public_html/wp-includes/plugin.php(476): WP_Hook->do_action(Array)
#6 /home/toi/public_html/wp-admin/admin-ajax.php(187): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-05-31 00:36:21 [warning X 4][/home/toi/public_html/wp-content/plugins/elementor/includes/managers/image.php::87] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 1 of 1JS: 2022-05-27 06:22:47 [error X 21][http://toi.dev.r2az.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.6.5:2:515479] Cannot read properties of null (reading \'getBoundingClientRect\') 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	Ultimate Addons for Elementor: Compatible

== Elementor Pro - Compatibility Tag ==
	
	Ultimate Addons for Elementor: Compatible

**Comment by rami-elementor** [comment]
@RichardNesbitt 

I tested this issue, you are correct, it's a bug.

But, activating "Optimized Markup" solves this issue, therefor the bug probably won't be prioritized and won't be fixed before the optimized markup experiment is merged.
